syncer/checkpoint.go

```diff
@@ -443,34 +443,32 @@ func (cp *RemoteCheckPoint) DeleteSchemaPoint(tctx *tcontext.Context, sourceSche
 	return nil
 }
 
-// IsNewerTablePoint implements CheckPoint.IsNewerTablePoint.
-// gte means greater than or equal, gte should judge by EnableGTID and the event type
-// - when enable GTID and binlog is DML, go-mysql will only update GTID set in a XID event after the rows event, for example, the binlog events are:
-//   - Query event, location is gset1
-//   - Rows event, location is gset1
+// IsOlderThanTablePoint implements CheckPoint.IsOlderThanTablePoint.
+// For GTID replication, go-mysql will only update GTID set in a XID event after the rows event, for example, the binlog events are:
+//   - Query event e1, location is gset1
+//   - Rows event e2, location is gset1
 //   - XID event, location is gset2
-//   after syncer handle query event, will save table point with gset1, and when handle rows event, will compare the rows's location with table checkpoint's location in `IsNewerTablePoint`, and these two location have same gset, so we should use `>=` to compare location in this case.
-// - when enable GTID and binlog is DDL, different DDL have different GTID set, so if GTID set is euqal, it is a old table point, should use `>` to compare location in this case.
-// - when not enable GTID, just compare the position, and only when grater than the old point is newer table point, should use `>` to compare location is this case.
-func (cp *RemoteCheckPoint) IsNewerTablePoint(sourceSchema, sourceTable string, location binlog.Location, gte bool) bool {
+// We should note that e1 is not older than e2
+// For binlog position replication, currently DM will split rows changes of an event to jobs, so some job may has save position.
+// if useLE is true, we use less than or equal.
```

Comment:
```suggestion
// if useLTE is true, we use less than or equal.
```

Reply:
most language use `LE` not `LTE` (seems a 4G terminology)

Reply:
https://en.wikipedia.org/wiki/Less-than_sign#Less-than_sign_plus_equals_sign